CONTRIBUTING.md

```diff
@@ -27,10 +27,30 @@ We are very happy to receive contributions from the community in any form!
 
 Please use a GitHub pull request to submit your contributions. If you have a
 question or are unsure if a contribution is wanted, please join us in
-[TBD](#channel-name-here) on Matrix to discuss your change or on the Ansible forum 
-using the TBD tag if you prefer async discussion. 
+[TBD](#channel-name-here) on Matrix to discuss your change or on the Ansible forum
+using the TBD tag if you prefer async discussion.
 Open a GitHub issue to report bugs or request features.
 
+## Running tests
+
```

Comment:
Should we add a line for xmlsec problem fix? Not sure if it's worth to mention it here